rpc/rpcserver/server.go

```diff
@@ -139,6 +141,201 @@ func decodeAddress(a string, params *chaincfg.Params) (dcrutil.Address, error) {
 type versionServer struct {
 }
 
+// ticketBuyerServer provides ticket buyer services for RPC clients.
+type ticketBuyerServer struct {
+	loader         *loader.Loader
+	ticketbuyerCfg *ticketbuyer.Config
+}
+
+// StartTicketPurchase enables ticket purchases in the ticket buyer.
```

Comment:
I actually don't require comments on these methods since they are invisible to godoc users.  all documentation should be placed in the api specification markdown document.